app/javascript/mastodon/features/emoji/unicode_to_filename.js

```diff
@@ -0,0 +1,26 @@
+// taken from:
```

Comment:
I took this from twemoji but it doesn't seem to work 100% of the time. I really don't understand how twemoji builds the SVG files